views/biological_information.py

```diff
@@ -1,10 +1,22 @@
 from rest_framework import generics, permissions
 from rest_framework.exceptions import NotFound
+from rest_framework.response import Response
+from rest_framework import status
+from django.http import HttpResponseBadRequest
+import json
 
 from omniport.utils.switcher import load_serializer
 
 
-class BiologicalInformationView(generics.RetrieveUpdateAPIView):
+def date_formatter(date):
```

Comment:
I don't know what exactly this function does. But you can checkout python's [datetime](https://docs.python.org/3/library/datetime.html) library to play with dates and even times.